graylog2-web-interface/src/theme/GlobalThemeStyles.jsx

```diff
@@ -345,14 +325,6 @@ const GlobalThemeStyles = createGlobalStyle(({ theme }) => css`
     margin-top: 5px;
   }
 
-  .system-system dt {
-    float: left;
-  }
-
-  .system-system dd {
-    margin-left: 75px;
-  }
-
```

Comment:
Love seeing all this red in the global styles file

Reply:
Me too 😉